src/components/content/order/orderStatus/useServiceDetailsPollingQuery.ts

```diff
@@ -17,3 +22,20 @@ export function useServiceDetailsPollingQuery(
         enabled: uuid !== undefined,
     });
 }
+
+export function useServiceDestroyDetailsPollingQuery(
+    uuid: string,
+    isDestroyDetailsQuerying: boolean,
+    refetchUntilStates: ServiceDetailVo.serviceDeploymentState[]
+) {
+    return useQuery({
+        queryKey: ['getServiceDetailsById', uuid],
+        // eslint-disable-next-line @typescript-eslint/no-non-null-assertion
```

Comment:
@Alice1319 i think we do not need this comment. There is no non-null assertion in the next line.